src/common/enum/CaptureModes.h

```diff
@@ -33,6 +35,26 @@ enum class CaptureModes
     Portal
 };
 
+static QString captureModeString(CaptureModes captureMode)
```

Comment:
I think it's good to not duplicate the translations but I don't like on this approach is that the translations will be in a "QObject" section and if you have multiple of them they go all in the same section but are actually in different files. How about having an enum translator class with a method for every enum type that we need to translate. The methods could have all the same name, like toString but differ in the parameter. If that class inherits from QObject, all translations will land in the same section that is called like the class itself.